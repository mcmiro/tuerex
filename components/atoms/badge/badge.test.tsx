/* eslint-disable */
import * as React from 'react';
import { screen, render } from '@testing-library/react';
import Badge from './';

describe('Test badge component', () => {
  it('Should pass when value is greater than 0', () => {
    const BADGE_VALUE = 1;
    render(
      React.cloneElement(<Badge value={BADGE_VALUE} data-testid="badge-test" />)
    );

    const renderBadgeComponent = screen.queryByTestId('badge-test');
    expect(renderBadgeComponent).toBeDefined();
    expect(renderBadgeComponent).toMatchSnapshot();
  });
});
