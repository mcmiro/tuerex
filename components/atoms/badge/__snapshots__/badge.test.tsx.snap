// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test badge component Should pass when value is greater than 0 1`] = `null`;
